src/site/content/en/blog/defining-core-web-vitals-thresholds/index.md

```diff
@@ -0,0 +1,486 @@
+achieves our criteria?"
+
+
+## Choice of percentile
+
+As noted earlier, to classify the overall performance of a page or site, we use
+the 75th percentile value of all visits to that page or site. The 75th
+percentile was chosen based on two criteria. First, the percentile should ensure
+that a majority of visits to a page or site experienced the target level of
+performance. Second, the value at the chosen percentile should not be overly
+impacted by outliers.
+
+These goals are somewhat at odds with one another. To satisfy the first goal, a
+higher percentile is typically a better choice. However, with higher
+percentiles, the likelihood of the resulting value being impacted by outliers
+also increases. If a few visits to a site happen to be on flaky network
+connections which result in excessively large LCP samples, we don't want our
+site classification to be decided by these outlier samples. For example, if we
+were evaluating the performance of a site with 100 visits using a high
+percentile such as the 95th, it would take just 5 outlier samples for the 95th
+percentile value to be affected by the outliers.
+
+Given these goals are a bit at odds, after analysis, we concluded that the 75th
+percentile strikes a reasonable balance. By using the 75th percentile, we know
+that most visits to the site (3 of 4) experienced the target level of
+performance or better. Additionally, the 75th percentile value is less likely to
+be affected by outliers. Returning to our example, for a site with 100 visits,
+25 of those visits would need to report large outlier samples for the value at
+the 75th percentile to be affected by outliers. While 25 of 100 samples being
+outliers is possible, it is much less likely than for the 95th percentile case.
+
+## Largest Contentful Paint
+
+### Quality of experience
+
+1 second is often cited as the amount of time a user will wait before they begin
+to lose focus on a task. On closer inspection of relevant research, we found
+that 1 second is an approximation to describe a range of values, from roughly
+several hundred milliseconds to several seconds.
+
+Two commonly cited sources for the 1 second threshold are [Card et
+al](https://dl.acm.org/doi/10.1145/108844.108874) and
+[Miller](https://dl.acm.org/doi/10.1145/1476589.1476628). Card defines a
+1-second "immediate response" threshold, citing Newell's [Unified Theories of
+Cognition](https://dl.acm.org/doi/book/10.5555/86564). Newell explains immediate
+responses as "responses that must be made to some stimulus within _very
+approximately one second_ (that is, roughly from ~0.3sec to ~3sec)." This
+follows Newell's discussion on "real-time constraints on cognition", where it is
+noted that "interactions with the environment which evoke cognitive
+considerations take place on the order of seconds" which range from roughly 0.5
+to 2-3 seconds. Miller, another commonly cited source for the 1 second
```

Comment:
```suggestion
to 3 seconds. Miller, another commonly cited source for the 1 second
```
At first glance this looked like a typo and then I realized what we were getting at. Since we're just mentioning a rough range it seems like we should just mention the outer bounds of the range?

Reply:
I agree this is awkward, but I think I'll keep it as is since I believe it was taken from the research.